web/src/components/tasks/business-formation/business/NonprofitProvisions.tsx

```diff
@@ -0,0 +1,169 @@
+import { Content } from "@/components/Content";
+import { BusinessFormationTextField } from "@/components/tasks/business-formation/BusinessFormationTextField";
+import { WithErrorBar } from "@/components/WithErrorBar";
+import { BusinessFormationContext } from "@/contexts/businessFormationContext";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useFormationErrors } from "@/lib/data-hooks/useFormationErrors";
+import { camelCaseToSentence } from "@/lib/utils/cases-helpers";
+import { FormationFields, FormationTextField } from "@businessnjgovnavigator/shared";
+import { FormControl, FormControlLabel, FormHelperText, Radio, RadioGroup } from "@mui/material";
+import { ReactElement, ReactNode, useContext } from "react";
+
+type RadioValues = "true" | "false" | "IN_FORM" | "IN_BYLAWS";
```

Comment:
[pebble] I don't love this type definition, there is no radio button that has these 4 values. We should have 2 types describing the 2 different types of radio buttons. You can still use the same functions like `getRadio`, it will probably just need to take a generic type so that the caller can specify which type to use

Reply:
(Thinking about this from the perspective of our tech talk we watched a while ago ["Making impossible states impossible"](https://www.youtube.com/watch?v=IcgmSRJHu_8))

Reply:
Updated